src/deployment/yearn/YearnDeployment.s.sol

```diff
@@ -24,6 +24,16 @@ contract YearnDeployment is BaseDeployment {
         return address(bridge);
     }
 
+    function approveAsset(address _bridge, address _asset) public returns (address) {
```

Comment:
both `approveAsset` and `approveAssets` exist here but in two different contexts? Can we refactor `approveAssets` to use `approveAsset` under the hood? and have the upstream deployment call `approveAsset` for both `eth` and `dai`. 

Having both will be harder to maintain / reason about when onboarding contributors,